src/items/item.cpp

```diff
@@ -2642,13 +2642,12 @@ std::string Item::getDescription(const ItemType &it, int32_t lookDistance, std::
 					s << "Vol:" << volume;
 				}
 			}
-			if (attack != 0) {
+			if (it.abilities && it.abilities->elementType != COMBAT_NONE && it.abilities->elementDamage != 0) {
```

Comment:
For correct display:

if (it.abilities && it.abilities->elementType != COMBAT_NONE && it.abilities->elementDamage != 0) {
				begin = false;
				if (attack != 0) {
					s << " (Atk:" << attack << " physical + " << it.abilities->elementDamage << ' ' << getCombatName(it.abilities->elementType);
				} else {
					s << " (" << it.abilities->elementDamage << ' ' << getCombatName(it.abilities->elementType);
				}
			} else if (attack != 0) {
				begin = false;
				s << " (Atk:" << attack;
			}

Reply:
In line 1184:

} else if (!it.isRanged()) {
			if (it.abilities && it.abilities->elementType != COMBAT_NONE && it.abilities->elementDamage != 0) {
				ss.str("");
				if (attack != 0) {
					ss << attack << " physical + " << it.abilities->elementDamage << ' ' << getCombatName(it.abilities->elementType);
				} else {
					ss << it.abilities->elementDamage << ' ' << getCombatName(it.abilities->elementType);
				}
				descriptions.emplace_back("Attack", ss.str());
			} else {
				descriptions.emplace_back("Attack", std::to_string(attack));
			}
		}

Reply:
I didn't find where to add a similar correction for the market data

Reply:
Fixed! Could you please check it again?